include/dem/particle_point_line_broad_search.h

```diff
@@ -17,8 +17,8 @@
  * Author: Shahab Golshan, Polytechnique Montreal, 2019
```

Comment:
```suggestion
```

Reply:
I suggest we do a PR removing all the author names in DEM so we will stop commenting about it haha